smarts/core/sumo_traffic_simulation.py

```diff
@@ -88,6 +88,9 @@ def __init__(
         self._debug = debug
         self._scenario = None
         self._log_file = None
+        assert (
+            time_resolution
+        ), "cannot use SUMO traffic simulation with variable time deltas"
```

Comment:
Yeah, I assume you're right. I didn't even look into this because we had deferred dealing with external traffic for now, b/c of our plan to refactor all of the traffic code very soon.  I honestly just didn't want to take the time to test it right now.
But if you think we can safely remove the `assert`, I'm fine with that.

Reply:
My preference is that it be removed but for the immediate case we can leave it in.